deps.versions.toml

```diff
@@ -6,12 +6,12 @@ essenty = "1.2.0-alpha-03"
 parcelizeDarwin = "0.1.4"
 reaktive = "1.2.3"
 junit = "4.13.2"
-jetbrainsCompose = "1.4.0"
+jetbrainsCompose = "1.5.0-beta01"
 jetbrainsKotlinWrappers = "1.0.0-pre.478"
 jetbrainsKotlinxCoroutines = "1.6.4"
 jetbrainsBinaryCompatibilityValidator = "0.12.1"
-jetpackCompose = "1.4.0"
-jetpackComposeCompiler = "1.4.5"
+jetpackCompose = "1.5.0-beta03"
```

Comment:
Perhaps we can now use Jetpack Compose `1.5.0`?

Reply:
Jetbrains Compose `1.5.0-beta02` makes use of Compose (Runtime, UI and more)`1.5.0`, so I updated both.